project-two/css/style.css

```diff
@@ -0,0 +1,334 @@
+@import url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+/* the suggested font face from the README.md is completely incorrect so... */
+@font-face {
+  font-family: "Avenir LT W01 35 Light";
+  src: url("../fonts/avenir-lt-w01-35-light.woff2");
+  /* this was the closest i could find to the original, down to the exact name, but its not perfect */
+}
+@font-face {
+  font-family: "Avenir LT W01 85 Heavy";
+  src: url("../fonts/avenir-lt-w01-85-heavy.woff2");
+  /* this was the closest i could find to the original, down to the exact name, but its not perfect */
+}
+
+/* CSS VARIABLES FOR THE COLOUR PALLETTE */
+:root {
+  --dark-beige: #E6DACD;
+  --light-beige: #F4ECE6;
+  --royal-blue: #0150FD;
+  --black: #000;
+  --white: #FFF;
+  --dark-gray: #5C5752;
+  --red: #FF4040;
+  --shadow: rgba(138, 131, 124, 0.23);
+}
```

Comment:
Good job on using css vars! Choosing names is tricky `--royal-blue` could be better as `--colour-primary` so that its value can be changed to orange while still making sense